### DISCUSSION TITLE
Exporting using Jinja?

### DISCUSSION THREAD

**Discussion by contagon** [mealie-recipes/mealie#573] [Q&A] [answered]
I've tried looking through the documentation a few times, but haven't found anything. Could someone give me a step by step on how to export recipes using the jinja templates? I can't find anywhere where to input the jinja "code"!

**Comment by hay-kot** [comment, marked as answer]
Any file in the templates directory can be used during a backup to generate the file. There's no way to do this through the UI at the moment so you'd have to transfers the files manually to the templates directory. Than it should show up in the list of options when backing up. 

It doesn't seem like a super popular feature so I haven't done a lot with it. If you find any problems with it, let me know!
